starknet-foundry/crates/forge/tests/integration/common/runner.rs

```diff
@@ -164,3 +164,30 @@ macro_rules! assert_failed {
         }));
     }};
 }
+
+#[macro_export]
+macro_rules! assert_case_output_contains {
+    ($result:expr, $test_case_name:expr, $asserted_msg:expr) => {{
+        use forge::test_case_summary::TestCaseSummary;
+
+        let test_case_name = $test_case_name;
+        let test_name_suffix = format!("::{test_case_name}");
```

Comment:
It is a bit error prone (two test functions can have the same name), but it is the best we can come up with for now I think

Reply:
you can do `module::name` and it will still work